Resources/Prototypes/Roles/Jobs/Medical/medical_intern.yml

```
@@ -17,11 +17,11 @@
   access:
   - Medical
   - Maintenance
-  #special: # Shitmed change  - begin DeltaV
+  #special: DeltaV - role modifiers bad
```

Add back the `# Shitmed change` so it reads `# Shitmed change. DeltaV - role modifiers bad`
```suggestion
  #special: # Shitmed change. DeltaV - role modifiers bad
```